README.md

```diff
@@ -1,8 +1,36 @@
 # Documentation Conventions Tests
 
-## Summary
+## What it is
+This is a test suite for [Emender](https://github.com/emender/emender) framework. It consists of two tests: Documentation Guidelines Test and Word Usage Test. One tests documentation against the existing guidelines, checks spelling etc. Another one checks for correct word usage in specific context. The tests generate the results in multiple formats and give you an extensive summary.
 
-Author of atomic_typos.txt is Stephen Wadeley (thanks!)
+## How to run it
+To run the test suite locally, follow these steps.
+1. Download the repository to your local machine. You don't need to have the test and documentation in the same folder. In fact, it's advisable not to.
+2. Download the documentation files into a separate folder. This folder must include:
+	* results.master
+		* directory containing "publican.cfg" and possibly "master.adoc".
+	* publican.cfg
+		* config file for your documentation, should have a "mainfile" parameter with the name of the master file, e.g. "mainfile: master"
+		* master file is located in the "your_language/" subfolder, e.g. "en-US/"
+	* results.cwd
+		* file containing documentation directory
+3. Documentation folder might or might not include "results.includes". It specifies which files (apart from "master.adoc") should be included in the tests if any.
+4. Obviously, you also need the documentation itself.
+5. Before running the test make sure to install Aspell dictionary.	
```

Comment:
I'd add info about service setup. w/o services (like whitelist/blacklist...) this test would fail.

Reply:
@tisnik I added some explanation in the README, but those are now optional. So in case the service doesn't work or something else happens, you still have Aspell. :)